routes/users.js

```diff
@@ -19,6 +19,8 @@ router.get("/userId/:userId", users.getUserById);
 router.patch("/self", authenticate, userValidator.updateUser, users.updateSelf);
 router.get("/", userValidator.getUsers, users.getUsers);
 router.get("/self", authenticate, users.getSelfDetails);
+router.get("/set-super-user-access", authenticate, users.setSuperUserAccessLimiter);
```

Comment:
Can you change the route name to be more descriptive? 
as this is a GET request and route name is kind on weird